stock_move_planned_consumed_date/views/stock_picking_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Foodles (http://www.foodles.co).
+     @author Pierre Verkest <pierreverkest84@gmail.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//page[@name='operations']/field[@name='move_ids_without_package']/tree/field[@name='reserved_availability']"
+                position="after"
+            >
+                <field name="planned_consumed_date" optional="hide" />
+            </xpath>
+            <xpath
+                expr="//page[@name='operations']/field[@name='move_ids_without_package']/form//field[@name='reserved_availability']"
+                position="after"
+            >
+                <field name="planned_consumed_date" optional="hide" />
```

Comment:
```suggestion
                <field name="planned_consumed_date" />
```

Reply:
@petrus-v I would say it should be optional="show" instead as tree view can be very overcharged.